db/src/main/kotlin/edu/artic/db/AppDataManager.kt

```diff
@@ -216,19 +216,36 @@ class AppDataManager @Inject constructor(
      * [primary stuff][getBlob]. As a side-effect, the injected [serviceProvider]
      * (and not `this` manager) is responsible for determining precisely
      * how many exhibitions and events we parse when this method is called.
+     *
+     * Observers on this should implement `onError` to handle I/O errors. We
+     * use [Observable.onErrorReturn] here to make sure that even if multiple
+     * errors occur at a low-level, only one call would go out to that `onError`.
      */
     private fun loadSecondaryData(): Observable<Int> {
         return Observable.zip(
```

Comment:
This can be changed to Observables.zip

Reply:
I think that _might_ be out of scope. We'll come back for that when we need to change the loading screen animation (part of AIC-17).